journey/pepr-deploy.ts

```diff
@@ -73,7 +73,7 @@ export function peprDeploy() {
       await until(() => state.done)
 
       // completes only if conditions are met, so... getting here means success!
-    }, 5000);
+    }, 10000);
```

Comment:
Blewup on me at 5 secs, but worked at 10 secs.  Looks like (at least on my machine) it ran for just over 6 secs.

```
    ...
    should monitor the cluster for admission changes
      ✓ npx pepr monitor should display validation results to console (6239 ms)
    ...
```